docs/02.API-REFERENCE.md

```diff
@@ -5178,7 +5178,8 @@ You can get them by calling jerry_get_object_native_pointer later.
 *Note*: If native pointer was already set for the object, its value is updated.
 
 *Note*: If a non-NULL free callback is specified in the native type information,
-        it will be called by the garbage collector when the object is freed.
+        it will be called by the garbage collector when the object is freed and
+        this callback **can not** invoke API functions.
```

Comment:
Some remarks:
- Appending this note to the existing sentence with an "and" sounds a bit weird. I'd suggest to simply start a new sentence. "when the object is freed. This callback"
- "can not" -> cannot (alternatively: "must not"?!)
- The following unrelated sentence has a glitch. It *is* unrelated but perhaps couldn't hurt fixing now? "type info ~~is~~ always overwrites"